config/routes.rb

```diff
@@ -9,7 +9,7 @@
 
   resources :users, only: [:edit, :update] do
     get 'edit-account', to: 'users#account'
-    resource :portfolio, :controller => "users/portfolios" do
+    resource :portfolio, :controller => "users/portfolios", except: [:show] do
       get '/delete', to: "users/portfolios#delete"
```

Comment:
@drod1000 I know this was not the scope for this ticket, but as we discussed a few times, this `get "/delete"` route still looks very strange...